bin/ch-fromhost

```diff
@@ -140,7 +125,7 @@ queue_files () {
                 ;;
         esac
         if is_so "$f"; then
-            debug "found shared library: ${f}"
+            [ -z "$verbose" ] || debug "found shared library: ${f}"
```

Comment:
Aha, here's where that test went. It needs to go inside `debug`; part of the point of that function is to insulate the caller from needing to test `$verbose`.

Reply:
The current implementation is admittedly ugly but it keeps `debug` defined in base.sh without having to alter much else.

We could add `--verbose` to `parse_basic_args` and remove that argument handling, along with `-v`, from ch-fromhost. Thoughts?

Reply:
> We could add `--verbose` to `parse_basic_args` in `base.sh` and remove that argument handling, along with `-v`, from ch-fromhost. Thoughts?

Currently `parse_basic_args` either does nothing or prints something and exits. If we add an argument that does not cause an exit, that argument must be consumed. If it's the first argument in `$@` that's easy because we can `shift` it off, but I don't know what to do if it's in the middle. Recall that problem was nearly impossible in POSIX sh when we were working on `ch-build2dir`.

Reply:
But, it would work to expect the caller to deal with `--verbose` and set `$verbose` if needed.

Reply:
Hmm, yes it isn't clear to me how to gracefully handle the verbose argument and set the verbose variable at the POSIX shell base.sh level.

I tinkered with a kludge in which the variable was handled twice, once when setting `verbose` in base.sh and again in ch-fromhost where the case essentially does nothing (see recent commits). It works but is it pretty horrendous.

Perhaps ch-fromhost should just use its own defined debug function? I'll think about this.